src/main/webapp/app/entities/exercise-hint/exercise-hint.model.ts

```diff
@@ -0,0 +1,12 @@
+import { Exercise } from 'app/entities/exercise';
+
+export interface IExerciseHint {
```

Comment:
I am not a big fan of these interfaces. I know that jhipster generates them, however we typically try to avoid them and work with the actual classes. So please have a look at other domain entities and follow how we define them

Reply:
And get rid of IExerciseHint

Reply:
Ok, I get it. I'll remove them.